database/migrations/2022_02_17_152932_create_choices_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateChoicesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create("choices", function (Blueprint $table) {
+            $table->id();
+            $table->foreignId("word_id");
```

Comment:
no onDelete cascade here?

Reply:
I have added the foreign key to reference choice "word_id" to word table "id". 
I also added the `onDelete("cascade")`